WordPress/Jetpack/Resources/AppStoreStrings.po

```diff
@@ -13,6 +13,11 @@ msgstr ""
 "Last-Translator: \n"
 "Language-Team: \n"
 
+#. translators: The application name in the Apple App Store. Please keep the brand names ('Jetpack' and WordPress') verbatim. Limit to 30 characters including spaces and punctuation!
+msgctxt "app_store_name"
+msgid "Jetpack"
```

Comment:
I intentionally used an outdate value here, so that when we'll run the code freeze automation for the next version, we'll see this changed in the diff, according to the value set in #19083.

You can see this working correctly in https://github.com/wordpress-mobile/WordPress-iOS/commit/b90be740f8358cc3452fdd130b38364eebf79e59

Reply:
It might be worth adding a task for this in the release scenario.